src/main.ts

```diff
@@ -75,7 +78,10 @@ export type { PurchasesErrorExtra } from "./entities/errors";
 export type { Period, PeriodUnit } from "./helpers/duration-helper";
 export type { HttpConfig } from "./entities/http-config";
 export { LogLevel } from "./entities/log-level";
-export type { GetOfferingsParams } from "./entities/get-offerings-params";
+export type {
+  GetOfferingsParams,
+  OfferingKeyword,
```

Comment:
Maybe I’m wrong but I think the enum should be exported normally (not just the type)

Reply:
Good catch. Done.